rmw_fastrtps_cpp/src/rmw_client.cpp

```diff
@@ -20,10 +20,8 @@
 #include "rmw/rmw.h"
 
 #include "rosidl_typesupport_introspection_cpp/identifier.hpp"
-
```

Comment:
Nitpick: Change unrelated to this PR. As we have an empty line between all include statements that come from different packages, I think this should stay.